### PR TITLE
About message for OSS

### PR DESCRIPTION
**Pull request by sbettevy**
Closes #

Changing About message for OSS

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [ ] Tests pass

### CODE REVIEW

**Comment by sbettevy** [comment]
capture d ecran 2019-02-13 a 08 39 27

File: ui/src/loudml/components/AboutPanel.js [hunk 1]
```diff
@@ -27,7 +28,10 @@ const AboutPanel = ({
                     It's plug-and-play, so you can use it for other projects too.
                 </p>
                 <p style={AboutTextStyle}>
-                    Visit <a href='http://loudml.io' target='_loudml'>loudml.io</a> to download the free Community package and reap the benefits of ML within days.
+                    Contribute to the open source software development on <a href="https://github.com/regel" target="_github">GitHub</a>.
```
Comment (regel) [inline ui/src/loudml/components/AboutPanel.js]: Pardon, l'url est: https://github.com/regel/loudml

**Review by regel — CHANGES_REQUESTED** [review]
dsl, url a completer

File: ui/src/loudml/components/AboutPanel.js (hunk 1 above)
Comment (sbettevy) [reply ui/src/loudml/components/AboutPanel.js]: in next commit

**Review by regel — APPROVED** [review]

**Pull request merged**